compare_ll.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+
+# In-package imports
+import corpus_utils
+import bucketers
+
+def print_header(header):
+  print(f'********************** {header} ************************')
+
+def parse_profile(profile):
+  kargs = {}
+  for kv in profile.split(','):
+    k, v = kv.split('=')
+    kargs[k] = v
+  return kargs
+
+def print_word_likelihood_report(corpus, ll1, ll2, bucket_type='freq',
+                          freq_count_file=None, freq_corpus_file=None,
+                          label_corpus=None, label_set=None):
+    """
+    Print a report comparing the word log likelihood.
+
+    Args:
+    corpus: the corpus of words over which the likelihoods are computed
+    ll1: likelihoods corresponding to each word in corpus from the sys 1
+    ll2: likelihoods corresponding to each word in corpus from the sys 2
+    bucket_type: A string specifying the way to bucket words together to calculate average likelihood
+    freq_corpus_file: When using "freq" as a bucketer, which corpus to use to calculate frequency.
+    freq_count_file: An alternative to freq_corpus that uses a count file in "word\tfreq" format.
+    label_corpus: When using "label" as bucket type, the corpus containing the labels
+                  corresponding to each word in the corpus
+    label_set: the permissible set of labels when using "label" as a bucket type
+    """
+    bucketer = bucketers.create_word_bucketer_from_profile(bucket_type=bucket_type,
+                                                         freq_count_file=freq_count_file,
+                                                         freq_corpus_file=freq_corpus_file,
+                                                         label_set=label_set)
+
+    if type(label_corpus) == str:
+        label_corpus = corpus_utils.load_tokens(label_corpus)
+
+    if label_corpus is not None:
+        corpus = label_corpus
+
+    ll1_out = bucketer.calc_bucketed_likelihoods(corpus, ll1)
+    ll2_out = bucketer.calc_bucketed_likelihoods(corpus, ll2)
+
+    print(f'--- avergae word log likelihood by {bucketer.name()} bucket')
+    for bucket_str, l1, l2 in zip(bucketer.bucket_strs, ll1_out, ll2_out):
+      print("{}\t{:.4f}\t{:.4f}".format(bucket_str, l1, l2))
+    print()
+
+
+
+if __name__ == '__main__':
+
+    parser = argparse.ArgumentParser(
+        description='Program to compare MT results',
+    )
+    parser.add_argument('--corpus', type=str, dest='corpus',
+                        help='A path to a reference file over which the likelihoods are being computed/compared')
+    parser.add_argument('--ll1-file', type=str, dest='ll1_file',
```

Comment:
Could we make the structure of the arguments the same as `compare_mt.py` for consistency? i.e. `ref`, `ll1`, and `ll2` should be positional arguments. All others should be options.